### PR TITLE
fix(university-gateway): filters, new page, query changes

### PR DESCRIPTION
**Pull request by Ballioli**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/screens/UniversitySearch/AboutPage.tsx [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+        <>
+          <Hidden above="sm">
+            <Box marginBottom={4}>
+              <Accordion>
+                <AccordionItem id="uni_dropdown" label="Háskólar">
+                  <Box width="full" className={cn(styles.courseListContainer)}>
+                    <Box className={cn(styles.courseListContentContainer)}>
+                      <Text variant="eyebrow" color="blueberry600">
+                        {' '}
+                        {/* TODO Translations */}
```
Comment (berglindoma13) [inline apps/web/screens/UniversitySearch/AboutPage.tsx:163]: Þarf ekki að gera þetta?

**Review by berglindoma13 — APPROVED** [review]

File: apps/web/screens/UniversitySearch/AboutPage.tsx [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+            </Box>
+          </Hidden>
+          <Hidden below="md">
+            <Box width="full" className={cn(styles.courseListContainer)}>
+              <Box
+                width="full"
+                className={cn(styles.courseListContentContainer)}
+              >
+                <Text variant="eyebrow" color="blueberry600">
+                  {' '}
```
Comment (RunarVestmann) [inline apps/web/screens/UniversitySearch/AboutPage.tsx:201]: Perhaps this space here is unintentional

File: apps/web/pages/haskolanam/um-haskolanam-is.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import withApollo from '@island.is/web/graphql/withApollo'
+import { withLocale } from '@island.is/web/i18n'
+import AboutPage from '@island.is/web/screens/UniversitySearch/AboutPage'
+import { getServerSidePropsWrapper } from '@island.is/web/utils/getServerSidePropsWrapper'
+
```
Comment (RunarVestmann) [inline apps/web/pages/haskolanam/um-haskolanam-is.tsx:5]: is the '-is-' suffix meant to denote that this is the icelandic locale?

**Review by RunarVestmann — APPROVED** [review]

File: apps/web/pages/haskolanam/um-haskolanam-is.tsx (hunk 3 above)
Comment (Ballioli) [reply apps/web/pages/haskolanam/um-haskolanam-is.tsx:5]: No Its meant as "Um háskólanám.is"

**Pull request merged**